scylla-proxy/Cargo.toml

```diff
@@ -31,3 +31,4 @@ rand = "0.8.5"
 assert_matches = "1.5.0"
 ntest = "0.9.0"
 tracing-subscriber = { version = "0.3.14", features = ["env-filter"] }
+
```

Comment:
The last commit adds nothing useful to `scylla-proxy`, only an empty line.

Reply:
As for the reason why it used to work: I suppose that Cargo resolves dependencies for all crates in the workspace at once, and some other crate than scylla-proxy depends on tokio and enables the "signal" feature.